git-tar/function/ops.go

```diff
@@ -42,6 +42,25 @@ func parseYAML(pushEvent sdk.PushEvent, filePath string) (*stack.Services, error
 	return parsed, err
 }
 
+func fetchTemplates(filePath string) error {
+	templateRepos := []string{"https://github.com/openfaas/templates", "https://github.com/alexellis/node8-express-template"}
```

Comment:
Are you sure this is on my account still and not the incubator?

Reply:
Thank you for the review. The value was old, sorry. I've fixed it to the incubator one.